internal/lotus/wallet.go

```diff
@@ -0,0 +1,33 @@
+package lotus
+
+import (
+	"github.com/NodeFactoryIo/hactar-daemon/internal/lotus/requests/lotus"
+	"github.com/NodeFactoryIo/hactar-daemon/pkg/util"
+)
+
+type WalletService interface {
+	GetWalletDefaultAddress() (string, error)
+	GetWalletBalance(walletAddress string) (string, error)
+}
+
+type walletService struct {
+	client *Client
+}
+
+func (ws *walletService) GetWalletDefaultAddress() (string, error) {
+	response, err := ws.client.lotusNodeClient.Call(lotus.WalletDefaultAddress)
```

Comment:
Not related to PR but... Are we reporting this address somewhere? I think it should be in general miner info (as it is in design there) but we kinda forgot about it.

Reply:
I will create another task and PR for this :+1: